unittests/POSIX/Known_Failures

```diff
@@ -35,3 +36,578 @@ conformance-interfaces-sigwaitinfo-6-1.test
 conformance-interfaces-sigwaitinfo-7-1.test
 conformance-interfaces-sigwaitinfo-8-1.test
 conformance-interfaces-sigwaitinfo-9-1.test
+
+# these are disabled
```

Comment:
Why did these change behaviour when adding gvisor tests?

Reply:
Disabled tests are now considered to fail with error -73, so they need to be added in the known failures list